usdigitalresponse/usdr-gost
# Disable click tracking for magic links

### Ticket #3097
## Description
Adding the ses:no-track attribute to disable click tracking for magic links. 

## Screenshots / Demo Video
N/A

## Testing
Tested to see if the attribute was included on the email's payload.

Steps:
* `docker compose exec app yarn workspace server run send-debug-email` with emailType: login passcode
* Go to mailpit, and check the caught email. 
* Check the raw html to see if it has a `ses:no-track` attribute

### Automated and Unit Tests

For this, do we want to have an explicit unit test that checks for the presence of `ses:no-track` somewhere in the email's payload?

### Manual tests for Reviewer

## Checklist
